unstract/tool-registry/tool_registry_config/sample_registry.yaml

```diff
@@ -9,7 +9,6 @@ auth:
   username: <username>
 document: unstract-tool-registry
 tools:
-  # - docker:unstract/tool-resume-parser
-  - local:fileops
```

Comment:
@chandrasekharan-zipstack are we not using this now?

Reply:
This is a sample file meant for reference and we don't have a fileops tool now. Didn't want to mention that old tool here - we do support downloading the image from docker and obtaining it locally as well